config/migrations/2022/20220317100000-remove-bestuurseenheden-op-1075.sparql

```diff
@@ -1,19 +1,36 @@
-# Deletes 2 worships services and their attributes (id, address, ...)
-
 DELETE {
   GRAPH ?g {
-    ?bestuur ?p ?o.
-    ?o ?p2 ?o2.
-    ?o2 ?p3 ?o3.
+    ?representatieveOrgaan <http://www.w3.org/ns/org#linkedTo> ?bestuur .
```

Comment:
this line is not used but it's not affecting the query so remove it or leave it, as you like haha